src/app/sequence-editor/sequence-editor.component.html

```diff
@@ -18,17 +18,20 @@ <h3 class="sequence-section-title">All Cards</h3>
     <div class="sequence-sequence-list">
       <h3 class="sequence-section-title">Sequences</h3>
       <ol class="card-list">
-        <li>Sequence 1</li>
-        <li>Sequence 2</li>
-        <li>Sequence 3</li>
+        @for( item of _sequences; track _sequences){
+          <li>{{item.name}}</li>
+        }
       </ol>
+      <button mat-icon-button aria-label="add to sequence" class="circle-button" (click)="addSequence()">
```

Comment:
Can you please position and shrink the add button to the correct space

Reply:
I will add this update on to the next item that I am working on because I have to do the _sequence-editor sass to not overwrite your button style for navbar.